robottelo/test.py

```diff
@@ -238,11 +240,21 @@ def take_screenshot(self, testmethodname):
         testmethodname = "_".join(testmethodname)
         # Creates dir_structure depending upon the testmethodname.
         directory = testmethodname
-        if not os.path.exists("/tmp/screenshots/" + directory):
-            os.makedirs("/tmp/screenshots/" + directory)
-        self.browser.save_screenshot('/tmp/screenshots/{dir}/'
-                                     'screenshot-{date}.png'
-                                     .format(dir=directory, date=filename))
+        os_ver = self.server_name.split(".")[0].split("-")[2]
+        if not os.path.exists('{screenshot_dir}/{date}/{ver}/{dir}/'
+                              .format(screenshot_dir=self.screenshot_dir,
+                                      date=date, ver=os_ver, dir=directory)):
+            os.makedirs('{screenshot_dir}/{date}/{ver}/{dir}/'
+                        .format(screenshot_dir=self.screenshot_dir, date=date,
+                                ver=os_ver, dir=directory))
+        self.browser.save_screenshot('{screenshot_dir}/{date}/{ver}/{dir}/'
+                                     'screenshot-{dnt}.png'
+                                     .format(
+                                         screenshot_dir=self.screenshot_dir,
+                                         date=date, ver=os_ver,
+                                         dir=directory, dnt=filename,
```

Comment:
`dnt=datetime.now().strftime('%Y-%m-%d_%H_%M_%S')` in order to drop single usage variable.